### PR TITLE
Add UK ami mappings

### PR DESCRIPTION
**Pull request by sevenmachines**
This change updates the available mapping to include eu-west-2
AMI's.

### CODE REVIEW

File: bootstrap_cfn/config.py [hunk 1]
```diff
@@ -1197,25 +1198,49 @@ def _get_os_data(self):
...
+              'windows2012': {
+                  'name': 'windows2012',
+                  'ami': 'ami-8519a9f6',
+                  'region': region,
+                  'distribution': 'windows',
+                  'type': 'windows',
+                  'release': '2015.12.31'
+              }
+          }
+        else:
```
Comment (mattpep) [inline bootstrap_cfn/config.py:1257]: It looks like the behaviour as proposed here is to mandate a value for `AWS_REGION`. Would it be better to have the default behaviour be to use `eu-west-1` - as that will preserve forwards compatibility?

Comment (sevenmachines) [reply bootstrap_cfn/config.py:1257]: Looking to centralise the region information in env.aws_region and synching the boto3.DEFAULT_SESSION to that too. but then defaulting the region to eu-west-1 as you mention for backwards compatibility.

Comment (sevenmachines) [reply bootstrap_cfn/config.py:1257]: Not sure I've done that yet though, might be a different PR, or sitting on the laptop somewhere

Comment (mattpep) [reply bootstrap_cfn/config.py:1257]: I've just found https://github.com/ministryofjustice/bootstrap-cfn/pull/241 which is the change you describe. (Linking to it here to keep the thread complete)

File: bootstrap_cfn/config.py [hunk 2]
```diff
@@ -1197,25 +1198,49 @@ def _get_os_data(self):
             OSTypeNotFoundError: Raised when the OS in the config file is not
                 recognised
         """
+        region=env.aws_region
         os_default = 'ubuntu-1404'
-        available_types = {
-            'ubuntu-1404': {
+        if region == 'eu-west-2':
+          available_types = {
+              'ubuntu-1604': {
```
Comment (ltsampros) [inline bootstrap_cfn/config.py:1205]: Shouldn't we have the corresponding image available in eu-west-1?

**Pull request merged**